ports/bext-di/portfile.cmake

```
@@ -1,8 +1,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO boost-ext/di
-    REF 6078f073feb8560127fbf0308b3794a551146252 # v1.2.0
-    SHA512 d7876b10b0365a92095ee7edefdc9cb9a0e237baf2bad124a927f22ed6ec30e18802b20517bc032184f39b7b4d71f44e965bdcea074646871d0194bd4e851d43
+    REF 28a356cc28877937bd29359ea5fa7daf04bd4d98 # v1.3.0
```

```suggestion
    REF "v${VERSION}"
```
We can use the new function.